src/alpine/README.md

```diff
@@ -7,7 +7,7 @@ Simple Alpine container with Git installed.
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | Alpine version: | string | 3.19 |
+| imageVariant | Alpine version: | string | 3.20 |
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!